tail_reader.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"io"
+	"os"
+	"time"
+)
+
+type tailReader struct {
+	io.ReadCloser
+}
+
+// Read provides a tailf like generator.
+//
+// The reader sleeps on EOF and is shutdown when the file is closed.
+// Implements the io.Reader inerface.
```

Comment:
This line belongs on the struct documentation.

Reply:
Fixed.